js/components/App.js

```diff
@@ -24,6 +38,8 @@ export default Relay.createContainer(App, {
             node {
               id,
               name,
+              enabled,
+              ${ToggleWidgetEnableMutation.getFragment('widget')},
```

Comment:
Wow, mutations really are complicated huh. Ok, thanks for the explanation @wincent !